### ISSUE TITLE
The repository size is too large

### ISSUE THREAD

**Issue by nacho00112** [bogdanfinn/tls-client#42]
You should consider reducing the repository size by some way, this isn't normal

**Comment by bogdanfinn** [comment]
@nacho00112 yes the problem is that back then i had the compiled binaries inside the repository because most of the users were not able to compile them alone.

i removed that and placed the binaries in the release section of github but they are still in the history... 
i would have to revisit every commit and manually remove the files from the history which would result in a ton of new commits and a possible force push ... 

See this SO answer for more information: https://stackoverflow.com/a/52643437

i'm not sure if i really should do it because this repo already has 53 forks ... 


maybe someone has a better idea/solution?

**Comment by sleeyax** [comment]
[git-extras](https://github.com/tj/git-extras) has an easy to use [git obliterate](https://github.com/tj/git-extras/blob/main/Commands.md#git-obliterate) sub-command that makes removal of junk trivial. It automatically rewrites the entire repo history for you. 

Unfortunately a force push is required and it will mess up all active forks, there's no way around that as far as I know.

**Comment by jlcd** [comment]
bump

this repo is still too large
